dali/pipeline/operators/reader/loader/indexed_file_loader.h

```diff
@@ -49,13 +49,27 @@ class IndexedFileLoader : public Loader<CPUBackend, Tensor<CPUBackend>> {
       current_file_ = FileStream::Open(uris_[file_index]);
       current_file_index_ = file_index;
     }
-    tensor->Resize({size});
-    tensor->mutable_data<uint8_t>();
 
-    int64 n_read = current_file_->Read(reinterpret_cast<uint8_t*>(tensor->raw_mutable_data()),
-                        size);
+    if (copy_read_data_) {
```

Comment:
Shouldn't it be `!copy_read_data_` here?

Reply:
You are right...

Reply:
Done